scalableminds/webknossos
# restored link to "view" dataset in gallery view

Datsets could not longer be loaded in"view" mode from the dataset gallery view. I restored the link for the "eye" icon and title cased some `title` attributes.

![image](https://user-images.githubusercontent.com/1105056/35809397-c7b8891a-0a88-11e8-8b6f-fd3e5b7333dc.png)


### Steps to test:
- Go to Dashboard --> Datasets
- Click on the "Eye" icon --> Dataset should load

------
